test/helpers/window.service.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
Regarding the test file, I didn't write it my self, a contributor who helped in testing a module of mine wrote this, and I just add it, I'm not comfortable with testings yet